spg/Animal/bin/barnyard.pl

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+
+use Animal;
+use Cow;
+use Horse;
+use Sheep;
+use Mouse;
+
+print "Build your own farm!\n";
+
+my @barnyard
```

Comment:
No statement ending semicolon? As this isn't the end of a block, I'm surprised that's allowed.